cli/admin/src/main/java/io/pravega/cli/admin/bookkeeper/BookkeeperDeleteLedgersCommand.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.bookkeeper;
+
+import io.pravega.cli.admin.CommandArgs;
+import lombok.Cleanup;
+import lombok.val;
+
+/**
+ * This command delete all the ledgers of the log starting with and including given ledger-id.
+ * It disables the log if enabled before deleting the ledgers.
+ * It can be used as last resort to recover the log in case recovery fails with missing ledger(s).
+ */
+public class BookkeeperDeleteLedgersCommand  extends BookKeeperCommand {
```

Comment:
We need a document in the `recovery-procedures` section of the documentation that explains how to get out from a situation in which a ledger is corrupted. Otherwise, we have this tool but not the scenarios in which it should be used and how to use it.

Reply:
You can add the document of this PR or maybe we can do it separately.

Reply:
yes, adding document is pending and I am thinking of adding it separately.